src/App.js

```diff
@@ -1,23 +1,29 @@
-import logo from './logo.svg';
 import './App.css';
+import { Routes, Route} from 'react-router-dom';
+import Header from './Header';
+import Creation from './Creation';
+import DisplayNewPoll from './DisplayNewPoll';
+import AllPolls from './AllPolls';
+import Footer from './Footer';
 
 function App() {
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+        <Routes>
```

Comment:
I love how you've organized your components into different paths on the landing page :)